src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -47,6 +48,8 @@ namespace winrt::TerminalApp::implementation
         DECLARE_EVENT_WITH_TYPED_EVENT_HANDLER(RequestedThemeChanged, _requestedThemeChangedHandlers, winrt::Windows::Foundation::IInspectable, winrt::Windows::UI::Xaml::ElementTheme);
 
     private:
+        bool _isUwp = false;

```

Comment:
```suggestion
        bool _isUwp{ false };
```

it's the only one that's not like the others ;P

Reply:
yep, sorry.